core/CPTs/EE_CPT_Strategy.core.php

```diff
@@ -126,7 +126,7 @@ private function _set_CPT_endpoints()
         $_CPT_endpoints = array();
         if (is_array($this->_CPTs)) {
             foreach ($this->_CPTs as $CPT_type => $CPT) {
-                $_CPT_endpoints [ $CPT['plural_slug'] ] = $CPT_type;
+                $_CPT_endpoints [ (string) $CPT['plural_slug'] ] = $CPT_type;
```

Comment:
should there be validation that there is a `'plural_slug'` index in the array as well?

Reply:
sure we could add that